MobileSyncExplorerReactNative/package.json

```diff
@@ -7,17 +7,17 @@
         "test": "jest"
     },
     "sdkDependencies": {
-        "SalesforceMobileSDK-iOS": "https://github.com/forcedotcom/SalesforceMobileSDK-iOS.git#dev",
-        "SalesforceMobileSDK-Android": "https://github.com/forcedotcom/SalesforceMobileSDK-Android.git#dev"
+        "SalesforceMobileSDK-Android": "https://github.com/wmathurin/SalesforceMobileSDK-Android.git#rn0701",
```

Comment:
Pointing to `wmathurin/SalesforceMobileSDK-Android.git#rn0701` for now allow CI to succeed.
Will point to `forcedotcom/SalesforceMobileSDK-Android.git#dev` before merging